api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -484,6 +484,10 @@ message HttpConnectionManager {
   // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
   // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
   bool merge_slashes = 33;
+
+  // The configuration of the extension to use for request ID utilities. This includes operations such as
+  // generation, validation, and associated tracing operations.
```

Comment:
Per my previous comment, I think if nothing is specified here we should just use the default implementation that is provided today? Can you clarify that?

Reply:
yeah I can clarify that, the current behaviour is that we use the UUIDUtils extension by default if nothing is provided. It is also linked in by default now.